### ISSUE TITLE
Static datasource images broken in multi_org setup

### ISSUE THREAD

**Issue by mangrose** [getredash/redash#5256]
### Issue Summary

Using the AWS AMI and successfully enabling multi_org support in Redash, I have found that the static images that illustrate datasources are broken. Looking at the source it looks like the culprit is in "client/app/services/data-source.js" and the IMG_ROOT variable. Adding a leading slash in the path would probably fix this.

### Steps to Reproduce

1. Enable multi_org support
2. Create datasource in organisation
3. All datasource images are broken

Static datasource images are shown correctly in a non multi_org setup and I think it should work in a multi_org setup as well.

### Technical details:

* Redash Version: Version: 9.0.0-beta (8d548ecb)
* Browser/OS: Chrome/MacOS
* How did you install Redash: AWS AMI

**Comment by kravets-levko** [comment]
Looks this issue is addressed in this PR: getredash/redash#5229

**Comment by mangrose** [comment]
Yes, looks like it. Thanks for the update. Would be good if it could be configurable as mentioned in the PR.

**Comment by keleyes** [comment]
you can use nginx solve this problem.
add in nginx configuration：
&nbsp;&nbsp; location ~^/.\*/static/images/  {
&nbsp;&nbsp;&nbsp;&nbsp; rewrite ^/.\*/static/images/(.*) &nbsp;&nbsp;  /static/images/$1;
&nbsp;&nbsp;}

**Comment by mangrose** [comment]
> you can use nginx solve this problem.
> add in nginx configuration：
>    location ~^/.*/static/images/ {
>      rewrite ^/.*/static/images/(.*)    /static/images/$1;
>   }

This worked perfectly, thanks!
